snap/info.go

```diff
@@ -135,6 +135,7 @@ type SideInfo struct {
 	EditedSummary     string   `yaml:"summary,omitempty" json:"summary,omitempty"`
 	EditedDescription string   `yaml:"description,omitempty" json:"description,omitempty"`
 	Private           bool     `yaml:"private,omitempty" json:"private,omitempty"`
+	Paid              bool     `yaml:"paid,omitempty" json:"paid,omitempty"`
```

Comment:
I wonder if this means `this snap is for sale` or `this snap was purchased`; Perhaps we could have a comment here.

Reply:
it means both/either actually depending on context